airflow-core/docs/start.rst

```diff
@@ -43,6 +43,36 @@ This quick start guide will help you bootstrap an Airflow standalone instance on
    This guide will help you quickly set up Apache Airflow using ``uv``, a fast and modern tool for managing Python environments and dependencies. ``uv`` makes the installation process easy and provides a
    smooth setup experience.
 
+   If you are on Windows, you have to use WSL2 (Linux environment for Windows).
+      .. code-block bash
```

Comment:
This is now formatted as comment in .rst (it will not be visible in the rendered page).